Makefile

```diff
@@ -275,7 +275,7 @@ test/version/version: version/version.go
 
 .PHONY: codespell
 codespell:
-	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
+	codespell -S bin,vendor,.git,go.sum,.cirrus.yml,"RELEASE_NOTES.md,*.xz,*.gz,*.ps1,*.tar,swagger.yaml,*.tgz,bin2img,*ico,*.png,*.1,*.5,copyimg,*.orig,apidoc.go" -L hastable,te,clos,ans,pullrequest,uint,iff,od,seeked,splitted,marge,erro,hist,ether,specif -w
```

Comment:
Why this addition? I can't find the string "hastable" anywhere in `main`; I can't even find it in `git log -S`

Reply:
codespell wants to change HasTable -> HashTable.
./cmd/podman/containers/ps.go:		noHeading = noHeading || !report.HasTable(listOpts.Format)
./cmd/podman/images/list.go:		if cmd.Flags().Changed("format") && !report.HasTable(listFlag.format) {